src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -255,12 +259,16 @@ foam.CLASS({
                   })
                 .end());
             }
-            spot.start('span').start('img').addClass(self.myClass('left-arrow'))
-                .attr('src', 'images/arrow-back-24px.svg')
-                .on('click', function() {
-                  self.carouselCounter--;
-                })
-              .end().end();
+            self.callIf(self.totalNumCards > self.MAX_NUM_DISPLAYBLE_CARDS, function() {
+              return spot.start('span')
```

Comment:
Is the span needed?

Reply:
@kgrgreer yes. We need the arrow to be an inline element so that it is correctly positioned.